whylog/client/const.py

```diff
@@ -1,2 +1,2 @@
 class BufsizeConsts:
```

Comment:
It was decided to use CamelCase here. See: https://github.com/9livesdata/whylog/pull/42#discussion_r54855438